review.lua

```diff
@@ -532,11 +532,39 @@ function Span(s, attr)
 end
 
 function RawInline(format, text)
-  return text
+  if (format == "review") then
+    return text
+  end
+
+  if (metadata.hideraw) then
+    return ""
+  end
+
+  if (format == "tex") then
+    return format_inline("embed", "|latex|" .. text)
+  elseif (format == "html") then
+    return format_inline("embed", "|html|" .. text)
+  else
+    return format_inline("embed", text)
```

Comment:
以下のように一般化してはどうでしょう?
対応しないビルダでもとりあえず通す感じです。

```suggestion
  else
    return format_inline("embed", "|" .. format .. "|", text)
```